laravel/resources/views/layouts/footer-main.blade.php

```diff
@@ -0,0 +1,7 @@
+<div class="container mt-auto">
+    <footer class="d-flex flex-wrap justify-content-between align-items-center py-3 my-4 border-top">
+        <div class="col-md-6 d-flex align-items-center">
+            <span class="text-muted">&copy; 2023 by <a href="https://github.com/Sebbo94BY" target="_blank">Sebbo94BY</a> | Make code with love ❤️❤️❤️</span>
```

Comment:
[SUGGESTION]

What about something like this? :-)

```suggestion
            <span class="text-muted">
                        &copy; Carbon::now()->format('Y') by <a href="https://github.com/Sebbo94BY" target="_blank">Sebbo94BY</a> and <a href="https://github.com/Sebbo94BY/teamspeak-dynamic-banner/graphs/contributors" target="_blank">Contributors</a> |
                        Powered by <i class="fa-brands fa-html5"></i> <i class="fa-brands fa-php"></i> <i class="fa-brands fa-laravel"></i> <i class="fa-brands fa-bootstrap"></i> <i class="fa-solid fa-font-awesome"></i> <i class="fa-brands fa-github"></i> |
                        <a href="https://github.com/Sebbo94BY/teamspeak-dynamic-banner/issues/new/choose" target="_blank">Open feature request or report issue</a>
            </span>
```